msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -152,7 +152,7 @@ The following table provides the user types and what features each user can acce
 | :-- | :-- | :-- | :-- | :-- | :-- | :-- |
 | Anonymous user | Not available | Not available | Not available | Interactions in the meeting chat are allowed. | Interactions in the meeting chat from an Adaptive Card are allowed. | Not available |
 | Guest that is part of the tenant AAD | Interaction is allowed. Creating, updating, and deleting are not allowed. | Not available | Not available | Interactions in the meeting chat are allowed. | Interactions in the meeting chat from an Adaptive Card are allowed. | Available |
-| Federated | Not available | Not available | Not available | Not available | Not available | Not available |
+| Federated | Available | Not available | Not available | Not available | Not available | Not available |
```

Comment:
For Federated Users, these are the values: 

Tabs - Interaction is allowed. Creating, updating, and deleting are not allowed.
Bot - Interaction is allowed. Acquiring, updating, and deleting are not allowed.
Message Extension - Not available 
Adaptive Cards - Interactions in the meeting chat are allowed.
Task Modules - Interactions in the meeting chat from an Adaptive Card are allowed.
In-meeting dialog - Not available 

Also, add this link for federated users so that you can navigate saying "Learn More here" - https://docs.microsoft.com/en-us/microsoftteams/non-standard-users